### PR TITLE
Update constants.ts  [bug] "Onlook is not enabled" -> "Learn how to e…

### PR DESCRIPTION
**Pull request by ScubyG**
### Description

Updated link to correct [bug] "Onlook is not enabled" -> "Learn how to enable" invalid link Issue #303

### What is the purpose of this pull request? 

- [ ] New feature
- [X] Documentation update
- [ ] Bug fix
- [ ] Refactor
- [ ] Release
- [ ] Other

### CODE REVIEW

File: app/common/constants.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export enum MainChannels {
 export enum Links {
     DISCORD = 'https://discord.gg/hERDfFZCsH',
     GITHUB = 'https://github.com/onlook-dev/onlook',
-    USAGE_DOCS = 'https://github.com/onlook-dev/onlook?tab=readme-ov-file#using-your-own-react-project',
+    USAGE_DOCS = 'https://github.com/onlook-dev/onlook?tab=readme-ov-file#option-1-use-your-own-react-project',
```
Comment (Kitenite) [inline app/common/constants.ts:66]: Could we use this link instead? https://github.com/onlook-dev/onlook/wiki/How-to-set-up-my-project%3F

**Review by ScubyG — COMMENTED** [review]
Changed wiki link for USAGE_DOCS

File: app/common/constants.ts (hunk 1 above)
Comment (ScubyG) [reply app/common/constants.ts]: I have changed the link it to https://github.com/onlook-dev/onlook/wiki/How-to-set-up-my-project%3F

**Review by Kitenite — APPROVED** [review]
Sonic speed!

**Pull request merged**